WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -278,6 +278,20 @@ def update_events
     end
   end
 
+  def update_wcif
+    @competition = competition_from_params
+    wcif = params.permit!.to_h
```

Comment:
I've updated the code to have an `update_wcif_events` endpoint instead of a general `update_wcif` endpoint. Let me know what you think of the new code, and we can discuss Lady Gaga if necessary =)